modules/install-sno-monitoring-the-installation-with-the-assisted-installer.adoc

```diff
@@ -0,0 +1,26 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="monitoring-the-installation-with-the-assisted-installer_{context}"]
+= Monitoring the installation with the Assisted Installer
+
+If you created the ISO using the Assisted Installer, use this procedure to monitor the installation.
+
+.Procedure
+
+. On the administration host, return to the browser and refresh the page. If necessary, reload the link:https://console.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer] page and select the cluster name.
+
+. Click *Next* until you reach step 3.
```

Comment:
Suggestion: Instead of referring to step numbers here, would it be possible to refer to the page in the installer? For example:
"Click *Next* until you reach step 3, _Networking_." 
or
"Click *Next* to _Review and Create_."